File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/host/exception/NotAcceptedHostException.java
package band.gosrock.domain.domains.host.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class NotAcceptedHostException extends DuDoongCodeException {
    public static final DuDoongCodeException EXCEPTION = new NotAcceptedHostException();

    private NotAcceptedHostException() {
        super(HostErrorCode.NOT_ACCEPTED_HOST);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/host/exception/NotAcceptedHostException.kt
package band.gosrock.domain.domains.host.exception

import band.gosrock.common.exception.DuDoongCodeException

class NotAcceptedHostException private constructor() : DuDoongCodeException(HostErrorCode.NOT_ACCEPTED_HOST) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = NotAcceptedHostException()
    }
}
